apps/docs/content/components/LogoSuite/index.mdx

```diff
@@ -0,0 +1,152 @@
+    </Caption>
+  </Dont>
+</DoDontContainer>
+
+### Color uniformity
+
+<DoDontContainer stacked>
+  <Do>
+    <img
+      src="https://github.com/primer/brand/assets/13340707/3245bcd5-4304-4d84-872e-6d30b30427d6"
+      alt=""
+    />
+    <Caption>
+      Ensure that the colors are consistent across the logos to match the logo
+      bar option. Ensure spacing between logos are consistent.
+    </Caption>
+  </Do>
+  <Dont>
+    <img
+      src="https://github.com/primer/brand/assets/13340707/a81f3423-b708-45c8-affc-d3d45a4b7cd8"
+      alt=""
+    />
+    <Caption>
+      Don’t use logos with inconsistent colors or don’t match the logo bar
+      option. Don’t use inconsistent spacing.
+    </Caption>
+  </Dont>
+</DoDontContainer>
+
+## Options
+
+### Logo bar appearance
+
+The logo suite offers three appearance options:
+
+- `muted`: The muted option is suitable for displaying 6+ logos and displays logos in a `secondary` color. It allows for two rows of logos, maintaining a clean and organized appearance.
+- `emphasis`: The emphasis option is designed for spotlighting ~5 logos in the `primary` color. It displays logos larger, drawing more attention to the logos.
+- `marquee`: The marquee option provides a flowing logo animation and can accommodate up to 12 logos. It is ideal for scenarios with limited page real estate.
+
+The muted and emphasis options on the logo bar contains an optional divider. This divider can serve as a visual separator. These three options can be used alongside one another when logo suite components are used adjacently.
+
+<div>
+  <img
+    src="https://github.com/primer/brand/assets/13340707/5ad51788-25a4-401a-91b0-02ec7407aa31"
+    alt=""
+  />
+  <Caption>
+    The muted option is the default logo bar. It is the best option for creating
+    2 rows (when using 6-12 logos).
+  </Caption>
+</div>
+
+<div>
+  <img
+    src="https://github.com/primer/brand/assets/13340707/6f239722-cb7b-4976-8782-09afbb1879b7"
+    alt=""
+  />
+  <Caption>
+    The muted option is the default logo bar. It is the best option for creating
+    2 rows (when using 6-12 logos).
+  </Caption>
+</div>
+
+<div>
+  <img
+    src="https://github.com/primer/brand/assets/13340707/d2dd578c-fce8-4999-ae07-066a616653ac"
+    alt=""
+  />
+  <Caption>The marquee options allows for different speed options.</Caption>
+</div>
+
+### Alignment
+
+The logo suite comes with start and center align options. Only the heading and description will have visible changes to their alignment as logos will be consistently center aligned to create balance. By default, the logo suite is aligned to the start of the page.
+
+## Accessibility
+
+- Always ensure that a heading is provided to the logo suite. Headings provide leading context to the logos shown, and serve as a navigation hook for assistive technologies. It can be visually hidden if needed.
+
+## Related components
+
+- [Image](/components/Image): to display a graphical representation.
```

Comment:
```suggestion
- [Image](/components/Image): to display a graphical representation.
- [Ordered List](/components/OrderedList): Use the ordered list component to display a list of numbered items.
- [Unordered List](/components/UnorderedList): Use the unordered list component to present a collection of items without specific ordering.
```

Reply:
👍 done

(I excluded those originally as I wasn't sure how relevant they were. No strong opinions on it though, so added them in)

Reply:
Sounds good! Thank you 😄 Actually Josep mentioned to add those lists, I also don't have a strong opinion and wasn't sure how relevant, thank you for adding 💯